test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/SortTests.cs

```diff
@@ -762,6 +762,34 @@ public async Task GivenPatientWithManagingOrg_WhenSearchedWithOrgIdentifierAndSo
                 patients.OrderByDescending(x => x.Name.Max(n => n.Family)).Cast<Resource>().ToArray());
         }
 
+        [SkippableFact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task GivenPatientWithManagingOrg_WhenSearchedWithOrgNameAndSortedByName_ThenPatientsAreReturned()
+        {
+            // Arrange Patients with linked Managing Organization
+            var tag = Guid.NewGuid().ToString();
+
+            var org = Samples.GetDefaultOrganization().ToPoco<Organization>();
+            org.Identifier.Add(new Identifier("http://e2etest", tag));
+            org.Name = "MSFT";
+            var orgResponse = await Client.CreateAsync(org);
+
+            var patient = Samples.GetDefaultPatient().ToPoco<Patient>();
+            patient.ManagingOrganization = new ResourceReference($"{KnownResourceTypes.Organization}/{orgResponse.Resource.Id}");
+
+            var patients = new List<Patient>();
+
+            SetPatientInfo(patient, "Seattle", "Robinson", tag);
+            patients.Add((await Client.CreateAsync(patient)).Resource);
+
+            SetPatientInfo(patient, "Portland", "Williams", tag);
+            patients.Add((await Client.CreateAsync(patient)).Resource);
+
+            string query = $"organization.name=MSFT&_sort=name";
+            var response = await Client.SearchAsync(ResourceType.Patient, query);
+            Assert.NotNull(response);
```

Comment:
I would suggest to add more assert validations on this test, per example: 
- Is the response returning all expected types?
- Is the response status code 200? 
- Are the results properly sorted by name?

Reply:
Added code to check all these. Also added another test case of orgs.